js/rolls.js

```diff
@@ -560,6 +560,16 @@ class PickupRoll extends ModifiedD6SumRoll {
   }
 }
 
+class BoneHeadRoll extends ModifiedD6SumRoll {
```

Comment:
For BoneHead, I think I'd probably give passing a value of 0, and failing a value of player TV * 2 half turns (because you lose use of that player for that turn and your opponents turn).

Because the effect on the summary is based on actual value vs expected value, passing a BoneHead roll will give you positive net value (because the expected value is negative, and your actual value was 0).